main/src/library/BigDecimal.flix

```diff
@@ -14,8 +14,50 @@
  * limitations under the License.
  */
 
+
+/// Represents arbitrary-precision signed decimal numbers.
+/// The representation consists of both a "value" and a "scale". Standard
+/// equality (`==` and the `Eq` typeclass) consider both value and scale,
+/// numerical equality (are the values the same after the scales have been
+/// normalized) is provided by the function `numericEquals`.
 namespace BigDecimal {
 
+    ///
+    /// Returns the smaller of `x` and `y`.
+    ///
+    pub def min(x: BigDecimal, y: BigDecimal): BigDecimal = if (x <= y) x else y
+
+    ///
+    /// Returns the larger of `x` and `y`.
+    ///
+    pub def max(x: BigDecimal, y: BigDecimal): BigDecimal = if (x >= y) x else y
+
+    ///
+    /// Returns the absolute value of `x`.
+    ///
+    pub def abs(x: BigDecimal): BigDecimal =
+        import java.math.BigDecimal.abs(): ##java.math.BigDecimal \ {};
```

Comment:
Does it not work here to write the type as `BigDecimal` instead of `##java.math.BigDecimal`?